### ISSUE TITLE
a11y check needed for `solr_helpers`

### ISSUE THREAD

**Issue by nichgray** [CDRH/lewisandclark#239]
The L&C views draw on https://github.com/CDRH/solr_helpers. I've made a few changes in the `a11y_fixes` branch of this to fix pagination, so this should be updated on all solr sites.

**Comment by techgique** [comment]
Updated across all three Rails sites which use `solr_helpers`
